ios/Podfile

```diff
@@ -34,4 +34,9 @@ target 'HathorMobile' do
     react_native_post_install(installer)
     __apply_Xcode_12_5_M1_post_install_workaround(installer)
   end
+
+  pod 'Firebase', :modular_headers => true
```

Comment:
Here https://firebase.google.com/docs/ios/setup#available-pods we have this text:

> For apps that use CocoaPods, the Firebase pod is deprecated in v9.0 and higher. Instead, you need to reference product pods directly in your Podfile (for example, FirebaseCore instead of Firebase/Core and FirebaseFirestore instead of Firebase/Firestore).

Do you need both `Firebase` and `FirebaseCore`?

Reply:
I was having issues installing the project on arm:

```bash
LoadError - dlopen(/Library/Ruby/Gems/2.6.0/gems/ffi-1.15.5/lib/ffi_c.bundle, 0x0009): tried: '/Library/Ruby/Gems/2.6.0/gems/ffi-1.15.5/lib/ffi_c.bundle' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64'))
```

To make it work, I had to install using `x86_64`:

```bash
arch -x86_64 pod install
```

But just checked, and removing this dependency (`Firebase`) solves the issue

Reply:
Done here [e8970](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/260/commits/e8970832089160e3eb1263c7d3aa35d1b0181e6d)